homeassistant/components/here_travel_time/sensor.py

```diff
@@ -0,0 +1,285 @@
+"""Support for HERE travel time sensors."""
+from datetime import datetime, timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_LATITUDE, ATTR_LONGITUDE, LENGTH_METERS, CONF_MODE, CONF_NAME,
+    EVENT_HOMEASSISTANT_START)
+from homeassistant.helpers import location
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
```

Comment:
please add `attribution` here

Reply:
I could not find any information on attribution required when using the here API like it is the case for Waze. Here itself uses third party services which may need attribution but this information is only available in the response as described here: https://developer.here.com/documentation/routing/topics/resource-type-source-attribution.html
I can include this as an attribute.

Reply:
`ATTRIBUTION` is for home assistant. It's not required but would greatly help for the travel time lovelace card

Reply:
Implemented